Running 2 test cases...
Entering test module "EmailDownloader_test"
../src/EmailDownloader_test.cpp(86): Entering test case "validateEmailSubject"
../src/EmailDownloader_test.cpp(104): info: check std::get<0>(EmailDownloader::validateEmailSubject(test2, configEmail.allowedSendersList.at(0))) has passed
../src/EmailDownloader_test.cpp(105): info: check std::get<0>(EmailDownloader::validateEmailSubject(test2, configEmail.allowedSendersList.at(2))) == false has passed
../src/EmailDownloader_test.cpp(86): Leaving test case "validateEmailSubject"; testing time: 417us
../src/EmailDownloader_test.cpp(110): Entering test case "validateEmailAgainstPrivileges"
../src/EmailDownloader_test.cpp(169): info: check messages.at(0).isValidated() has passed
../src/EmailDownloader_test.cpp(170): info: check messages.at(1).isValidated() has passed
../src/EmailDownloader_test.cpp(172): info: check 1694258520ULL == messages.at(0).getValidUntil() has passed
../src/EmailDownloader_test.cpp(173): info: check 1694258520ULL == messages.at(1).getValidUntil() has passed
../src/EmailDownloader_test.cpp(110): Leaving test case "validateEmailAgainstPrivileges"; testing time: 362us
Leaving test module "EmailDownloader_test"; testing time: 837us
